scripts/xsct.tcl

```diff
@@ -12,6 +12,19 @@ if {$m_mode == "init"} {
   sdk createhw -name hw -hwspec $hdf_file
   sdk createapp -name xilsw -hwproject hw -proc $cpu_name -bsp bsp -app {Hello World}
   sdk createapp -name sw -hwproject hw -proc $cpu_name -bsp bsp -app {Empty Application}
+  if { $cpu_name == "sys_mb" } {
```

Comment:
isn't there some project settings somewhere, where this can be modified without doing linker tricks ?